### PR TITLE
Added endpoints for skill reviewing system.

### PR DESCRIPTION
**Pull request by dynamitechetan**
Progress on issue #436 
Added API endpoints to Save Skills to a unreviewed folder and another endpoint for getting all skills saved in that unreviewed skills folder.
image
<br>
image

### CODE REVIEW

**Comment by dynamitechetan** [comment]
only users can create the skills and only users/admins can review them. That's why I chose base user role of USER.

File: src/ai/susi/server/api/cms/GetUnreviewedSkills.java [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+ * This Endpoint accepts 1 parameter i.e model name. Default is "general"
+ * can be tested at:
+ * http://127.0.0.1:4000/cms/getUnreviewedSkills.json
+ */
+public class GetUnreviewedSkills extends AbstractAPIHandler implements APIHandler {
+
+    private static final long serialVersionUID = -8691003678852307876L;
+
+    @Override
+    public BaseUserRole getMinimalBaseUserRole() { return BaseUserRole.ANONYMOUS; }
```
Comment (saurabhjn76) [inline src/ai/susi/server/api/cms/GetUnreviewedSkills.java]: Please change it to User

Comment (dynamitechetan) [inline src/ai/susi/server/api/cms/GetUnreviewedSkills.java]: Done

Comment (dynamitechetan) [inline src/ai/susi/server/api/cms/GetUnreviewedSkills.java]: Done

**Comment by mariobehling** [comment]
Low priority now. Therefore closing it.

**Pull request closed without merging**